### PR TITLE
Support old import system in POT

### PR DESCRIPTION
**Pull request by l-bat**
### Details:
 - *Added support for legacy imports*
```python
from compression.api import DataLoader as POTDataLoader
from compression.api import Metric
from compression.engines.ie_engine import IEEngine
from compression.graph import load_model, save_model
from compression.graph.model_utils import compress_model_weights
from compression.pipeline.initializer import create_pipeline
```
 - *Enabled import  `DataLoader, Metric, IEEngine, load_model, save_model, compress_model_weights, create_pipeline` without `api` dir*
 ```python
from openvino.tools.pot import DataLoader, Metric, IEEngine, load_model, save_model, compress_model_weights, create_pipeline
```

### Tickets:
 - *74012*

### CODE REVIEW

**Review by AlexKoff88 — APPROVED** [review]

**Comment by AlexKoff88** [comment]
@l-bat, have you checked that it works from setup.py/wheel?

File: tools/pot/compression/__init__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import sys
+
+import openvino.tools.pot.api
+import openvino.tools.pot.engines.ie_engine
+import openvino.tools.pot.graph
+import openvino.tools.pot.graph.model_utils
+import openvino.tools.pot.pipeline.initializer
+
+
+sys.modules["compression.api"] = openvino.tools.pot.api
```
Comment (AlexKoff88) [inline tools/pot/compression/__init__.py:10]: can we show a message that this API is deprecated?

File: tools/pot/openvino/tools/pot/__init__.py [hunk 2]
```diff
@@ -33,6 +33,13 @@
 from .algorithms.quantization.overflow_correction.algorithm import OverflowCorrection
 from .algorithms.quantization.ranger.algorithm import Ranger
 
+from .api.data_loader import DataLoader
+from .api.metric import Metric
+from .engines.ie_engine import IEEngine
```
Comment (AlexKoff88) [inline tools/pot/openvino/tools/pot/__init__.py:38]: "Engine" should be also imported

**Review by AlexKoff88 — APPROVED** [review]

**Review by KodiaqQ — APPROVED** [review]

**Review by helena-intel — APPROVED** [review]
Thanks for adding this!

**Pull request merged**